scripts/actions/translation_workflow/database.js

```diff
@@ -154,6 +154,18 @@ const addTranslationsJobsRecord = async (translationId, jobId) => {
   return record;
 };
 
+/**
+ * Method to get records from 'translations_jobs' table.
+ * @param {number} jobId - identifier of job
+ */
+const getTranslationsJobsRecords = async (jobId) => {
```

Comment:
this works, but id say copy the same setup as the other get methods 🙈  where an object is accepted, and jobid (or translationId) can be a field on the object used to get results.

Reply:
@nr-kkenney you are always biased 😂 I can't remember why i deviated from the others but i'm sure it was based on something 😛 i'll update that now